pkg/ebpf/netlink/conntracker.go

```diff
@@ -20,6 +20,8 @@ import (
 
 const (
 	initializationTimeout = time.Second * 10
+
+	compactInterval = time.Minute * 5
```

Comment:
Chatted in person but I think it's safer to have this <= the expiry interval, so 4 minutes would be good.

Reply:
Done

Reply:
just curious, was the motivation to not "miss" a generation expiration in case the compaction call is delayed?

Reply:
It would otherwise take ~20 hours to roll around and eventually clean up the record.

Reply:
That was exactly the case @p-lambert :)